Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -61,6 +62,15 @@ GameFile::GameFile(const QString& path) : m_path(path)
   }
 
   m_valid = true;
+
+  static Core::TitleDatabase title_database;
```

Comment:
Yep. For reference, DolphinWX keeps a `TitleDatabase` instance around in it's `GameListCtrl` and passes a reference to every `ISOFile` (I think this is the equivalent of `GameFile`).